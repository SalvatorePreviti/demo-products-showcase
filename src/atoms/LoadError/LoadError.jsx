import React from 'react'

function LoadError() {
  return (
    <div>
      <b>Loading error.</b> Try to reload
    </div>
  )
}

export default LoadError
