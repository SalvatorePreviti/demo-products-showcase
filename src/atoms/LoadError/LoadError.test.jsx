import React from 'react'
import { shallow } from 'enzyme'

import LoadError from './LoadError'

describe('LoadError', () => {
  it('snapshots', () => {
    expect(shallow(<LoadError />)).toMatchSnapshot()
  })
})
