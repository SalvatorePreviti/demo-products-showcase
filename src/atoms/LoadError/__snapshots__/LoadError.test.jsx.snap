// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadError snapshots 1`] = `
<div>
  <b>
    Loading error.
  </b>
   Try to reload
</div>
`;
